docs/api/en/materials/MeshBasicMaterial.html

```diff
@@ -93,7 +93,10 @@ <h3>[property:Float lightMapIntensity]</h3>
 		<p>Intensity of the baked light. Default is 1.</p>
 
 		<h3>[property:Texture map]</h3>
-		<p>The color map. Default is null.</p>
+		<p>
+			The color map. Default is null. <br />
+			To enable alpha channel transparency, [page:Material.transparent .transparent] also needs to be set to `true`.
```

Comment:
There are a few more options, like using the alpha channel with `.alphaTest` for clipping/masking. Perhaps:

> The color map. May optionally include an alpha channel, typically combined with [page:Material.transparent .transparent] or [page:Material.alphaTest .alphaTest]. Default is null.

Reply:
I'm fine with that version as well.

Reply:
@ggPeti Would updated the PR with @donmccurdy's suggested text? However, you have to update all material pages that contain a `map` documentation. Ideally you also copy the updated text over to the `zh` pages.